### PR TITLE
refactor: add new eslint rules and apply changes

### PR DESCRIPTION
**Pull request by EncryptedDev**

### CODE REVIEW

**Comment by Bas950** [comment]
@EncryptedDev @ImRodry wen ist zis rdy?

**Comment by EncryptedDev** [comment]
> @EncryptedDev @ImRodry wen ist zis rdy?

probably this weekend

**Comment by ImRodry** [comment]
Yeah, will resolve the issues and hopefully finish it by this weekend, we also gotta compile all of them and I can’t do all that *cough* @EncryptedDev

**Comment by EncryptedDev** [comment]
@Slowlife01 @Bas950 pls review

**Comment by ImRodry** [comment]
I tried to keep the commit history clean but rhys added a merge commit
Either way we made a massCompiler script that checks for errors you’re welcome

**Comment by ImRodry** [comment]
@Bas950 @Slowlife01 :)

**Comment by TheDropX** [comment]
@Bas950

**Comment by EncryptedDev** [comment]
@Bas950 @Slowlife01 now it is ready for review

File: websites/W/Wolvesville/presence.ts [hunk 1]
```diff
@@ -322,23 +314,24 @@ presence.on("UpdateData", async () => {
                 .querySelector(
                   "div.css-1dbjc4n.r-150rngu.r-eqz5dr.r-16y2uox.r-1wbh5a2.r-5oul0u.r-11yh6sk.r-1rnoaur.r-1sncvnh.r-13qz1uu"
                 )
-                ?.innerHTML.match(/text-decoration-line: line-through/g) || []
+                ?.textContent.match(/text-decoration-line: line-through/g) || []
```
Comment (down-bad) [inline websites/W/Wolvesville/presence.ts]: why did the `innerHTML`s changed to `textContent`s? 
in this case it breaks the presence.

Comment (EncryptedDev) [reply websites/W/Wolvesville/presence.ts]: textContent is better to use over innerHTML, maybe in this case it was not appropriate

**Comment by Slowlife01** [comment]
*On `no-one-time-vars` latest you can remove a few eslint-ignores*

**Comment by ImRodry** [comment]
That’s what I did on the latest commit but it seems like I forgot to bump it

**Comment by ImRodry** [comment]
@Bas950 @Slowlife01 pls

**Pull request merged**